### ISSUE TITLE
导入其他中文模型时遇到问题

### ISSUE THREAD

**Issue by Ealvn** [dpm12345/nonebot_plugin_tts_gal#17]
感谢大佬做了中文模型适配, 我在导入别人训练的中文模型时遇到了一些问题

1. 模型(用的是目录下G_209000.pth, 压缩包内包含项目文件): [下载地址](https://pan.baidu.com/s/1YvQVHPO8HfFYepufaHZuZA?psd=haya)
2. 配置文件
![QQ图片20221111165200](https://user-images.githubusercontent.com/39033468/201380143-38eb8797-a3e9-466c-abeb-b70cb1252fbb.png)
`{
  "train": {
    "log_interval": 200,
    "eval_interval": 1000,
    "seed": 1234,
    "epochs": 2000,
    "learning_rate": 2e-4,
    "betas": [0.8, 0.99],
    "eps": 1e-9,
    "batch_size": 24,
    "fp16_run": true,
    "lr_decay": 0.999875,
    "segment_size": 8192,
    "init_lr_ratio": 1,
    "warmup_epochs": 0,
    "c_mel": 45,
    "c_kl": 1.0
  },
  "data": {
    "training_files":"yousa/train_filelist.txt.cleaned",
    "validation_files":"yousa/val_filelist.txt.cleaned",
    "text_cleaners":["chinese_cleaners1"],
    "max_wav_value": 32768.0,
    "sampling_rate": 22050,
    "filter_length": 1024,
    "hop_length": 256,
    "win_length": 1024,
    "n_mel_channels": 80,
    "mel_fmin": 0.0,
    "mel_fmax": null,
    "add_blank": true,
    "n_speakers": 0,
    "cleaned_text": true
  },
  "model": {
    "inter_channels": 192,
    "hidden_channels": 192,
    "filter_channels": 768,
    "n_heads": 2,
    "n_layers": 6,
    "kernel_size": 3,
    "p_dropout": 0.1,
    "resblock": "1",
    "resblock_kernel_sizes": [3,7,11],
    "resblock_dilation_sizes": [[1,3,5], [1,3,5], [1,3,5]],
    "upsample_rates": [8,8,2,2],
    "upsample_initial_channel": 512,
    "upsample_kernel_sizes": [16,16,4,4],
    "n_layers_q": 3,
    "use_spectral_norm": false
  },
  "language":"haya"
}`

因为个人感觉原程序与大佬的差异很大, 所以没有在配置里添加符合表. 而是根据原项目, 对以下文件做了调整:
1. __init__.py
![QQ图片20221111164504](https://user-images.githubusercontent.com/39033468/201383890-f26f4809-b6bd-4085-a09e-7fda982da17d.png)
![QQ图片20221111164511](https://user-images.githubusercontent.com/39033468/201383891-058c10f9-db4e-4783-97ac-5b5a5d6c71f2.png)

2. text/init.py
![QQ图片20221111164514](https://user-images.githubusercontent.com/39033468/201384234-c2e5087d-0972-4ff0-a061-cfeb090f5c91.png)

3. text/cleaners.py
![QQ图片20221111164518](https://user-images.githubusercontent.com/39033468/201384445-4f695a00-6920-49db-9b86-2be3174befae.png)

4. text/symbol.py
![QQ图片20221111164522](https://user-images.githubusercontent.com/39033468/201384844-70ae1b4f-1f99-4090-a931-3e241b4db6f8.png)

最后还是会有报错...
![QQ图片20221111164200](https://user-images.githubusercontent.com/39033468/201384987-d246a018-2127-4045-8088-9b6ccfceac45.png)

所以想请教一下大佬, 是不是我思路不对, 考虑得太复杂了

**Comment by dpm12345** [comment]
在代码不出错应该是没问题的
看起来像是symbols和训练时有点不同
然后`language`为`haya`的话，自动翻译应该会出错

**Comment by Ealvn** [comment]
> 

想问下大佬, text/symbol.py里的符号表是不是大于等于训练模型时的符号表就可以

**Comment by dpm12345** [comment]
> > 
> 
> 想问下大佬, text/symbol.py里的符号表是不是大于等于训练模型时的符号表就可以


应该是要相等

**Comment by Ealvn** [comment]
emmm, 我太想当然了, 还是得好好了解vits才行, 谢谢大佬
